src/App.jsx

```diff
@@ -14,70 +15,71 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export default function App() {
+  const jam = goods.find(e => e === 'Jam');
+  const [item, setItem] = useState(jam);
```

Comment:
you can delete this `const jam = goods.find(e => e === 'Jam');`
and just wite `useState('Jam');`

Reply:
also give more clear name....what this item is?

replace with
` const [selectedItem, setSelectedItem] = useState(jam); `